pelion_test_lib/tools/manifest_tool.py

```diff
@@ -80,7 +80,11 @@ def create_manifest(path, firmware_url, update_image_path, output='output.manife
     if os.path.isfile(output):
         os.remove(output)
     if manifest_version == 'v1':
-        cmd = [MANIFEST_DEV_TOOL, 'create-v1', '-u', firmware_url, '-o', output]
+        if fw_version  is None:
+            cmd = [MANIFEST_DEV_TOOL, 'create-v1', '-u', firmware_url, '-o', output]
+        else:
+            cmd = [MANIFEST_DEV_TOOL, 'create-v1', '-u', firmware_url, '-o', output, '--fw-version', fw_version]
+
```

Comment:
```suggestion
        cmd = [MANIFEST_DEV_TOOL, 'create-v1', '-u', firmware_url, '-o', output]
```
Better to add it below [Line 92](https://github.com/PelionIoT/pelion-e2e-python-test-library/pull/27/files#diff-3815e1336aa6b1667bf1b3d8ded11a2a156427e7cd185c0f6e3a596ca1058c95R92) to include v3 too:
```
if fw_version:
    cmd += [ '--fw-version', fw_version ]
```